tools/jinja2/_compat.py

```diff
@@ -10,6 +10,8 @@
     :copyright: Copyright 2013 by the Jinja team, see AUTHORS.
     :license: BSD, see LICENSE for details.
 """
+# flake8: noqa
+
```

Comment:
suggesting to drop these based on my earlier comment.

Reply:
Excluded from new `lint-py` target:
https://github.com/nodejs/node/blob/4458162cfaf8c1eed987323fbb21c86783a8e914/Makefile#L1221-L1223